### DISCUSSION TITLE
couple server crashes from the new pvp game modes

### DISCUSSION THREAD

**Discussion by LePiromano** [FakeFishGames/Barotrauma#14224] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

tried the new pvp game modes, first a friend hosted the server, loaded in the outpost deathmatch mode, i nuked the fuck out of his subamrine, it killed him, and then i boarded his sub and killed his last and only bot, and then it crashed:
[servercrashreport.log](https://github.com/user-attachments/files/16021379/servercrashreport.log)

then i hosted my own server and tried the king of the hill mode, we load in, can't find the beacon so i go straight to him, i nuke the fuck out of his armory which blows up the ic-4 inside and it kills him, and the server crashes:
[servercrashreport.log](https://github.com/user-attachments/files/16021386/servercrashreport.log)

### Reproduction steps

i'm pretty sure in both of these what caused the crash is killing one of the enemy players, in both cases the setting was set to 104% to complete the objective

### Bug prevalence

Happens regularly

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.6.100.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Fixed in https://github.com/FakeFishGames/Barotrauma-development/commit/9bed8757311f001b9a22833fa41a57f248af8033, fix should be coming up in a new unstable build later today
